lib/mayaUsd/utils/utilFileSystem.cpp

```diff
@@ -179,9 +179,9 @@ std::string UsdMayaUtilFileSystem::getPathRelativeToProject(const std::string& f
     if (fileName.empty())
         return {};
 
-    MString projectPath = MGlobal::executeCommandStringResult("workspace -q -rd");
     // Note: don't use isEmpty() because it is not available in Maya 2022 and earlier.
-    if (projectPath.length() == 0)
+    const MString projectPath = UsdMayaUtil::GetCurrentMayaWorkspacePath();
+    if (projectPath.isEmpty())
```

Comment:
The comment right above says you cannot use MString::isEmpty().

Reply:
Oh my bad. I missed that comment..